packages/vite/src/node/ssr/ssrManifestPlugin.ts

```diff
@@ -40,6 +44,50 @@ export function ssrManifestPlugin(config: ResolvedConfig): Plugin {
               })
             }
           }
+          if (chunk.code.includes(preloadMethod)) {
+            // generate css deps map
+            const code = chunk.code
+            let imports: ImportSpecifier[]
+            try {
+              imports = parseImports(code)[0].filter((i) => i.d > -1)
+            } catch (e: any) {
+              this.error(e, e.idx)
+            }
+            if (imports.length) {
+              for (let index = 0; index < imports.length; index++) {
+                const { s: start, e: end, n: name } = imports[index]
+                // check the chunk being imported
+                const url = code.slice(start, end)
+                const deps: string[] = []
+                if (isQuoted(url)) {
```

Comment:
is this checking for static URL? I think we can determine this by the presence of `d` (dynamicIndex) instead of relying on quotes